infra/security.tf

```diff
@@ -65,7 +65,7 @@ module "iam_eks_role" {
   oidc_providers = {
```

Comment:
Name this module better instead of "iam_eks_role" as this module creates irsa only for Load Balancer, you will have more of this modules so they need to be unique in naming.